### PR TITLE
setup github actions

### PR DESCRIPTION
**Pull request by trepmal**
Addresses #35

### CODE REVIEW

File: bin/install-wp-tests.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```
Comment (mikeyarce) [inline bin/install-wp-tests.sh:1]: Should we use the latest scaffold tests?  It has a handy `[skip-database-creation]` paramater which I find useful sometimes. 
Latest: https://github.com/wp-cli/scaffold-command/blob/479cd4f10f638d7a46a5b4a60c80bd4b4a6bd5e2/templates/install-wp-tests.sh

**Comment by trepmal** [comment]
updated

File: bin/install-wp-tests.sh (hunk 1 above)
Comment (trepmal) [reply bin/install-wp-tests.sh:1]: updated

**Review by mikeyarce — APPROVED** [review]
Looks good!   Tested all the composer commands locally 👍🏽

**Pull request merged**